terraform/jenkins/agents.tf

```diff
@@ -23,41 +23,41 @@ resource "aws_security_group" "jenkins_agent" {
   tags = var.tags
 }
 
-resource "aws_instance" "jenkins_agent" {
-  count = var.agents_count
-
-  ami = var.agent_ami
-
-  subnet_id = module.vpc.private_subnets[count.index % length(module.vpc.private_subnets)]
-
-  instance_type = var.agent_instance_type
-
-  vpc_security_group_ids = [aws_security_group.jenkins_agent.id]
-
-  root_block_device {
-    volume_size = 100
-  }
-
-  key_name = var.ssh_key_name
-
-  user_data = templatefile("${path.module}/install_docker.sh", { jenkins_version = var.jenkins_version })
-
-  tags = merge(var.tags,
-    {
-      Name = "folio-jenkins-agent-${count.index + 1}"
-  })
-}
+# resource "aws_instance" "jenkins_agent" {
```

Comment:
@sergii-msn 

Could we put a reason why following resources are commented and when they should be uncommented or removed?

Reply:
those resources was created when we are not planned to use agents on k8s pods. And now we are not succesfull with those k8s pods, so we are not sure we do not need old approach anymore. I just comment it for now because we do not want to pay for those resources at this moment.